splitio/models/telemetry.py

```diff
@@ -41,6 +41,7 @@ class ConfigParams(Enum):
     EVENTS_QUEUE_SIZE = 'eventsQueueSize'
     IMPRESSIONS_MODE = 'impressionsMode'
     IMPRESSIONS_LISTENER = 'impressionListener'
+    FLAG_SETS = 'FlagSetsFilter'
```

Comment:
should start with lowecase?

Reply:
Correct,  the class (ConfigParams) should be lower case, will check and correct all the constant classes in Telemetry model.